src/adaptation/stepsize.jl

```diff
@@ -133,7 +133,10 @@ function adapt_stepsize!(
         @unpack m, ϵ, x_bar, H_bar = state
     end
 
-    @pack! state = m, ϵ, x_bar, H_bar
+    state.m = m
+    state.ϵ = 
+    state.x_bar = x_bar
+    state.H_bar = H_bar
```

Comment:
I plan to add a `@pack!` macro based on `setproperty` this week (possibly earlier), so in case you don't want the hand-written assignments you could wait a few days.